src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpDebuggerIntellisenseNetCore.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Microsoft.VisualStudio.IntegrationTest.Utilities;
+using Xunit;
+
+namespace Roslyn.VisualStudio.IntegrationTests.CSharp
+{
+    [Collection(nameof(SharedIntegrationHostFixture))]
+    public class CSharpDebuggerIntellisenseNetCore : CSharpDebuggerIntellisenseCommon
+    {
+        public CSharpDebuggerIntellisenseNetCore(VisualStudioInstanceFactory instanceFactory)
+            : base(instanceFactory, WellKnownProjectTemplates.CSharpNetCoreConsoleApplication)
+        {
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Debugger)]
+        public override void StartDebuggingAndVerifyBreakpoints()
+        {
+            base.StartDebuggingAndVerifyBreakpoints();
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Debugger)]
```

Comment:
In xUnit you can put the Facts directly in the base type, and it searches for those when running tests. That way you can delete all these forwarders and it's very nice and clean.